#! /bin/sh /usr/share/dpatch/dpatch-run
## 50_hp-clean_fix.dpatch by Henrique de Moraes Holschuh <hmh@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Fix wipeAndSpitType1() invocation

if [ $# -ne 1 ]; then
    echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
    exit 1
fi

[ -f debian/patches/00patch-opts ] && . debian/patches/00patch-opts
patch_opts="${patch_opts:--f --no-backup-if-mismatch}"

case "$1" in
       -patch) patch $patch_opts -p1 < $0;;
       -unpatch) patch $patch_opts -p1 -R < $0;;
        *)
                echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
                exit 1;;
esac

exit 0

@DPATCH@
diff -urNad hplip/clean.py /tmp/dpep.eqBpGV/hplip/clean.py
--- hplip/clean.py	2005-04-26 01:32:22.040450887 -0300
+++ /tmp/dpep.eqBpGV/hplip/clean.py	2005-04-26 01:45:49.943553532 -0300
@@ -178,7 +178,7 @@
 
     if clean_type == 1:
         if level == 3:
-            maint.wipeAndSpitType1()
+            maint.wipeAndSpitType1( d )
         elif level == 2:
             maint.primeType1( d )
         else:
@@ -194,7 +194,7 @@
     
     elif clean_type == 3:
         if level == 3:
-            maint.wipeAndSpitType1()
+            maint.wipeAndSpitType1( d )
         elif level == 2:
             maint.primeType1( d )
         else:
